src/compiler/crystal/tools/playground/public/vendor/codemirror-5.38.0/mode/crystal/crystal.js

```diff
@@ -30,7 +30,7 @@
     var keywords = wordRegExp([
       "abstract", "alias", "as", "asm", "begin", "break", "case", "class", "def", "do",
       "else", "elsif", "end", "ensure", "enum", "extend", "for", "fun", "if",
-      "include", "instance_sizeof", "lib", "macro", "module", "next", "of", "out", "pointerof",
+      "include", "instance_sizeof", "lib", "macro", "module", "next", "of", "offsetof", "out", "pointerof",
```

Comment:
This change should go against codemirror. It doesn't hurt to keep it, but I would change the version to `codemirror-5.38.0-offsetof` to hightlight that we are using a custom version.

Reply:
I don't think it's worth it making a custom change to this vendored file. We should just adapt the upstream defintion and updated accordingly.